kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class BoardConfig {
+
+    @Autowired
+    @Qualifier("taskToDoList")
+    TaskList toDoList;
+
+    @Autowired
+    @Qualifier("taskInProgressList")
+    TaskList inProgressList;
+
+    @Autowired
+    @Qualifier("taskDoneList")
+    TaskList doneList;
+
+    @Bean
+    @Scope("prototype")
```

Comment:
why do you decided to create all beans as a prototype?

Reply:
In this case I could change @scope of @Bean getBoard to singleton. Instance of TaskList I used more than once.